### DISCUSSION TITLE
Does presentation mode support images?

### DISCUSSION THREAD

**Discussion by YconquestY** [vuepress-theme-hope/vuepress-theme-hope#2412] [Q&A] [answered]
Hi, I'd like to insert images to my slides, but neither of the following attempts works

```markdown
---

…
![](img_path)
…

---
```

```markdown
---

…
<img src="img_path">
…

---
```

**May I know how to add images to presentation slides?**

**Comment by Mister-Hope** [comment, marked as answer]
## Maybe you NEED to carefully read through docs and demos.

https://vuepress-theme-hope.github.io/v2/guide/markdown/presentation.html#demo

![image](https://user-images.githubusercontent.com/33315834/201586787-b5f2c3c7-7d3d-44ae-bbc9-a90a518ca6c7.png)

https://vuepress-theme-hope.github.io/v2/md-enhance/guide/presentation/demo.html#/3/2

https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/main/docs/md-enhance/src/guide/presentation/demo.md?plain=1#L163

I don't think I do NEED any improvements.
